Machine_Learning/project.py

```
@@ -0,0 +1,115 @@
+# Load trained model
+model = pickle.load(open("D:/hs/model23.pkl", "rb"))  # Use relative path for portability
+
+# Get coordinates from pincode using OpenStreetMap
+def get_location_osm(pincode):
+    try:
+        url = f"https://nominatim.openstreetmap.org/search?postalcode={pincode}&country=India&format=json"
+        response = requests.get(url, headers={"User-Agent": "Mozilla/5.0"})
+        data = response.json()
+        if data:
+            return float(data[0]['lat']), float(data[0]['lon'])
+    except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)